cmd/vm-informant/main.go

```diff
@@ -80,8 +80,9 @@ func main() {
 		}
 
 		runRestartOnFailure(ctx, args, cleanupHooks)
-		// this cancels the context
-		srv.GetShutdown(ctx)()
+		closer := srv.GetShutdownSignal(ctx)
+		// this cancels the process' underlying context
+		closer()
```

Comment:
```suggestion
		cancel := srv.GetShutdownSignal(ctx)
		// this cancels the process' underlying context
		cancel()
```

Not sure if this is better, but a thought. Take it or leave it

Reply:
I think maybe it should be `shutdown`, but I'm indifferent.